test/expressions/array-expression.js

```diff
@@ -62,7 +62,19 @@ suite('Parser', function () {
 
   });
 
-  testParse('[a, ...(b=c)]', expr, {
```

Comment:
It shouldn't. It's valid code and continues to parse; it's just that the parentheses aren't really doing anything interesting except making the parser's job easier.